wadm-local.yaml

```diff
@@ -42,85 +42,10 @@ spec:
           properties:
             replicas: 1
 
-    - name: orange-vod-metaverse-agent
```

Comment:
Please use the wadm local file inside deployment folder and delete this one as it keeps re added every time

Reply:
I mean we should delete this one, and you will find `wadm-from-file.yml` inside the deployment folder, which should replace the one in the root folder